src/pymatgen/io/cif.py

```diff
@@ -969,8 +969,22 @@ def _get_structure(
         primitive: bool,
         symmetrized: bool,
         check_occu: bool = False,
+        min_thickness: float = 0.1,
```

Comment:
I hope this default minimal "thickness" of 0.1 Å would be a good default value, is there any case that we might see valid structure below this?